packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss

```diff
@@ -17,6 +17,7 @@
     color: #3d3d3d;
     display: block;
     margin: auto;
+    max-height: 120px;
```

Comment:
can we remove max-height from other container?

Reply:
good point removed  form parent.

I have question w.r.t callouts 
![image](https://user-images.githubusercontent.com/6234682/30135586-fefbad88-9320-11e7-9e67-fadf8691839f.png)
with this change we are overwriting the height to 120 px max. The strange part is callout doesn't have specific logo height for mobile so not sure from where 120 px came in. Or the callout may be outdated. For now I am keeping max height to 120 px and removing the height constraint from parent container (media query).